backend/functions/lib/controllers/case-assignment/case.assignment.controller.test.ts

```diff
@@ -11,6 +14,9 @@ import {
 import { CamsRole } from '../../../../../common/src/cams/roles';
 import { CamsUserReference } from '../../../../../common/src/cams/users';
 import { MANHATTAN } from '../../../../../common/src/cams/test-utilities/offices.mock';
+import { UnknownError } from '../../common-errors/unknown-error';
+import HttpStatusCodes from '../../../../../common/src/api/http-status-codes';
+import { buildTestResponseSuccess } from '../../../azure/testing-helpers';
```

Comment:
We should not be reaching into `/backend/functions/azure` from anywhere in `/lib`.